packages/react-native-icons/src/utils/FluentIconsProps.types.ts

```diff
@@ -1,8 +1,8 @@
-import * as React from "react";
+import type { ColorValue, TextProps } from "react-native";
+import type { SvgProps } from "react-native-svg";
 
-export type FluentIconsProps<TBaseAttributes extends (React.SVGAttributes<SVGElement> | React.HTMLAttributes<HTMLElement>) = React.SVGAttributes<SVGElement>> = (TBaseAttributes) & {
-	primaryFill?: string
-	className?: string
```

Comment:
Is there any way removing the className might cause problems for folks, or what is the likelihood it was being used by anyone? Alternately, is there a significant cost to leaving it in so as to prevent possible negative impact to users?

Reply:
Also we don't use classNames in RN, so removing them shouldn't be an issue. Styles get added via passing a property bag into a style prop for RN, we don't have CSS classNames we can use, unfortunately